dts/arm/renesas/ra/ra8/r7fa8m1ahecbd.dtsi

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/renesas/ra/ra8/r7fa8m1xh.dtsi>
+#include <zephyr/dt-bindings/clock/ra_clock.h>
+
+/ {
+	clocks {
```

Comment:
As I understand it, the only difference in the branch numbers is the flash size.
Wouldn't it be better to include the clock in `dts/arm/renesas/ra/ra8/ra8x1.dtsi`?

Reply:
dts/arm/renesas/ra/ra8/ra8x1.dtsi is target for RA8M1, RA8D1 and RA8T1. Clock will be different between them. We will consider moving to r7fa8m1xh.dtsi.

Reply:
We updated it. Please check. Thank you.